Home.html

```diff
@@ -14,64 +14,98 @@
     </head>
     <body>
         <main>
-            <section class="home p-2">
+            <section class="home p-2" id="home-section">
                 <nav class="navbar navbar-expand-sm">
-                    <div class="container-fluid">
+                    <div class="container-fluid hamburger-container">
                         <img class="hamburger-btn" src="assets/menu-icon-png-3-lines-10.jpg" alt="Hamburger menu">
                     </div>
                 </nav>
                     <h5>"Hello Travelers"</h5>
-                    <h1>Afghan<br>Travel Agency</h1>
-                    <div class="description container-fluid">
```

Comment:
- [ ] Kindly make sure your project title text is written in `uppercase` as for the Behance template design. I recommend you apply this on your both pages `Home` and `About` pages.